remoteit/desktop
# [DESK-723] context menu

### Description


Provide service context menus on device list.
Show scroll bars on hover.
Copy deep link from menu.
Other UI improvements.

### Test plan



1. Sign in
2. Go to device list
3. Click a service indicator
4. You should see a new menu
5. You should be able to copy a deep link

### Ticket



<https://remoteit.atlassian.net/browse/DESK-723>
